blocks/table-of-contents/src/script.js

```diff
@@ -1,7 +1,10 @@
 /**
  * Table of Contents
  */
-document.querySelectorAll( '.wp-block-cata-toc' ).forEach(handleBlock);
+const tocBlocksRendered = new CustomEvent( 'tocBlocksRendered' );
```

Comment:
Do we use this variable for anything else? Could line 7 be condensed to?

`document.dispatchEvent( new CustomEvent( 'tocBlocksRendered' ) )`